packages/expo-system-ui/ios/ExpoSystemUI/ExpoSystemUIModule.swift

```diff
@@ -3,22 +3,56 @@
 import ExpoModulesCore
 
 public class ExpoSystemUIModule: Module {
+  
+  public required init(appContext: AppContext) {
+    super.init(appContext: appContext)
+    
+    // TODO: Maybe read from the app manifest instead of from Info.plist.
```

Comment:
Yeah, I think in the future we can make the app manifest available from the app context. For now this solution seems fine.